asmcomp/s390x/emit.mlp

```diff
@@ -469,6 +469,14 @@ let emit_instr env i =
         `	brc	{emit_int mask}, {emit_label lbl}\n`;
         `	lghi	{emit_reg i.res.(0)}, 0\n`;
         `{emit_label lbl}:\n`
+    | Lop(Icompf cmp) ->
+        let lbl = new_label() in
+        `	cdbr	{emit_reg i.arg.(0)}, {emit_reg i.arg.(1)}\n`;
+        `	lghi	{emit_reg i.res.(0)}, 1\n`;
+        let mask = branch_for_float_comparison cmp in
+        `	brcl	{emit_int mask}, {emit_label lbl}\n`;
```

Comment:
You can use `brc` here (like above for integer comparisons), as you know the label will be very close.